gatsby/src/components/list-card/measure-list-card.module.scss

```diff
@@ -1,6 +1,44 @@
+@import '../../../assets/scss/design-system/base/variables';
+@import '../../../assets/scss/design-system/vendor/bootstrap/functions';
+@import '../../../assets/scss/design-system/vendor/bootstrap/mixins';
+@import '../../../assets/scss/design-system/vendor/bootstrap/variables';
+
 .measure-list-card-detail {
   font-size: 1.3rem;
   line-height: 2rem;
   font-weight: 500;
   color: #2362a2;
 }
+
+@include media-breakpoint-up(xl) {
```

Comment:
Not really sure about the breakpoint to be honest. Id probably go for something more narrow? Cause seeing mobile design at `1190px` feels weird :D Doesnt it?

Reply:
I totally agree with @rostaklein, it should be lg.

Reply:
Gotcha, let me change it. xl felt the most natural, but we can always easily tweak this when we add the left menu :)